test/services/audit/AuditServiceSpec.scala

```diff
@@ -107,10 +107,10 @@ trait AuditTestSupport extends UnitSpec with ExportsDeclarationBuilder with Scal
   val mockAuditConnector = mock[AuditConnector]
 
   val auditData = Map(
-    EORI.toString -> "eori1",
+    eori.toString -> "eori1",
     LRN.toString -> "lrn1", // FIXME refere to bad thing
     DUCR.toString -> "ducr1",
```

Comment:
Here you're using wrong labels, that contains probably correct value, but this LRN and DUCR are not from EventData

Reply:
Those are fine, here but I am going to explicitly refer to EvenData.